example-get-started-experiments/code/.github/workflows/run-experiment.yml

```diff
@@ -64,32 +65,27 @@ jobs:
           STUDIO_TOKEN: ${{ secrets.STUDIO_TOKEN }}
           DVCLIVE_LOGLEVEL: DEBUG
         run: |
-          dvc pull
           cml ci --fetch-depth 0
```

Comment:
No action needed, but we only need `--fetch-depth 0` for the report step below and will eventually drop it, right? I think another option would be to add `git fetch` at the start of the report step.

Reply:
> and will eventually drop it, right?

Yes, I hope to drop it soon in favor of a Studio-based one.